docs/udp.rst

```diff
@@ -433,3 +433,131 @@ which we can process further and download::
     cube.download("fancy.tiff", format="GTiff")
 
 See :ref:`datacube_from_process` for more information on :func:`~openeo.rest.connection.Connection.datacube_from_process`.
+
+
+.. _workflow_example:
+
+Working Example: Create EVI Timeseries UDP
+==========================================
+
+In this first example, we'll create an openEO workflow script,
+including declaring parameters, building, storing and loading a UDP
+
+Here, let us consider that we want to recalculate the EVI-timeseries
+as done in :ref:`example-use-case-evi-map-and-timeseries` but for a different area.
+Therefore for future, instead of redefining the whole process, we simply 
+want to define it as a UDP ``EVI_timeseries``, that can be easily shared 
+and reused later without dealing with the entire workflow each tim:
+
+.. code-block:: python
+
+    import openeo
+    from openeo.api.process import Parameter
+
+    # Create connection to openEO back-end
+    connection = openeo.connect("...").authenticate_oidc()
+
+    # declaring parameters
+    temporal_interval = Parameter(
+        name="temporal_interval",
+        description="The date range to load.",
+        schema={"type": "array", "subtype": "temporal-interval"},
+        default =["2018-06-15", "2018-06-27"]
+    )
+    bbox = Parameter(
+        name="bbox",
+        description="The bounding box to load.",
+        schema={"type": "object", "subtype": "geojson"},
+        default={
+            "west": 5.14,
+            "south": 51.17,
+            "east": 5.17,
+            "north": 51.19,
+            "crs": 4326,
+        }
+    )
+    feature = Parameter(
+        name="feature",
+        description="Provide polygon as FeatureCollection for extracting the EVI (Enhanced Vegetation Index) timeseries for these regions.",
+        schema={"type": "object", "subtype": "geojson"},
+        default={
+            "west": 5.14,
+            "south": 51.17,
+            "east": 5.17,
+            "north": 51.19,
+            "crs": 4326,
+        }
+    )
+
+    # Load raw collection data
+    sentinel2_cube = connection.load_collection(
+        "SENTINEL2_L2A",
+        spatial_extent = bbox,
+        temporal_extent = temporal_interval,
+        bands = ["B02", "B04", "B08", "SCL"],
+    )
+
+    # Extract spectral bands and calculate EVI with the "band math" feature
+    blue = sentinel2_cube.band("B02") * 0.0001
+    red = sentinel2_cube.band("B04") * 0.0001
+    nir = sentinel2_cube.band("B08") * 0.0001
+    evi = 2.5 * (nir - red) / (nir + 6.0 * red - 7.5 * blue + 1.0)
+
+    evi_aggregation = evi_masked.aggregate_spatial(
+        geometries = feature,
+        reducer = "mean",
+    )
+
+    # Store user-defined process in openEO back-end.
+    process_name = "EVI_timeseries"
```

Comment:
it's more standard to use "process_id" instead of "process_name"

Reply:
Also note that the standard processes don't user upper case in their id, so I would use process id `evi_timeseries` or even just `evi`

Reply:
noted and corrected. Didn't think about it; thank you for correcting me.